runtime/src/anchor/mod.rs

```diff
@@ -237,6 +253,17 @@ decl_module! {
 
             Ok(())
         }
+
+        // move_anchor dispatch call when anchor by anchor_id is to be moved to another chain
+        pub fn move_anchor(origin, anchor_id: T::Hash) -> Result {
+            // ensure signed origin
+            ensure_signed(origin)?;
```

Comment:
Is this origin the validator signature? or the client/requester signature?

Reply:
This validates the sender of the dispatch call. Ethereum does it out of the box but in substrate different accounts can send a dispatch call. For this function `ensure-signed` is enough. For sudo users, we would be using a different fucntion. 

@vimukthi-git can shed more light on this maybe ?

Reply:
This is basically the signer of the extrinsic, or the entity who originated the call. Ensure signed is used whenever we want to make sure that the extrinsic is signed. Signing account is the account which pays the transaction fees if any also.

Reply:
That means that the chain needs to know the public key to verify the signature. Is that a previous setup needed for each account? Or it is implicit like in Ethereum where public key is the address (simplifying)? Or the public key goes along the `origin` object?

Reply:
Its like ethereum i guess.

Reply:
Yup its like ethereum, account addresses are public keys.